pkg/jobcreator/controller.go

```diff
@@ -141,6 +148,7 @@ func (controller *JobCreatorController) subscribeToWeb3() error {
 	controller.web3Events.Storage.SubscribeDealStateChange(func(ev storage.StorageDealStateChange) {
 		deal, err := controller.solverClient.GetDeal(ev.DealId)
 		if err != nil {
+			err = fmt.Errorf("module allowlist error: %s", err)
```

Comment:
This looks off, if that's the error then it should bubble up from where it happened no? The way this code reads it says any error during the `GetDeal` process is a module allowlist err